src/codegen/templates/_library_interpreter.py.mako

```diff
@@ -27,6 +27,7 @@ class LibraryInterpreter(BaseInterpreter):
         self._done_event_callbacks = []
         self._every_n_samples_event_callbacks = []
         self._signal_event_callbacks = []
+        self._new_session_initialized = True
```

Comment:
`new_session_initialized` should be stored in the same place as the task handle. 

Because we are not storing the task handle in the interpreter object, one interpreter may create multiple tasks. Some of these tasks may already exist, and others may not.

Example:
```
system = nidaqmx.system.System.local() # creates interpreter
tasks = []
for persisted_task in system.tasks: # reuses interpreter
    tasks.append(persisted_task.load()) # reuses interpreter
```

I realize that `LibraryInterpreter.create_task` and `LibraryInterpreter.load_task` don't actually overwrite this field, but there is no reason for it to be a `LibraryInterpreter` field in the first place. These methods should either define it as a local variable or simply return a `True` literal. (The former is more readable, and I suspect that's what you were going for.)